core/trino-main/src/main/java/io/trino/type/UuidOperators.java

```diff
@@ -41,7 +40,7 @@ private UuidOperators() {}
     public static Slice uuid()
     {
         java.util.UUID uuid = randomUUID();
-        return wrappedLongArray(uuid.getMostSignificantBits(), uuid.getLeastSignificantBits());
+        return javaUuidToTrinoUuid(uuid);
```

Comment:
Is it important that `javaUuidToTrinoUuid` calls `reverseBytes` but the existing code did not?

Reply:
Good catch. i think @electrum would know.

Reply:
The original code is wrong.